modules/ROOT/pages/jakarta-ee10-diff.adoc

```diff
@@ -63,6 +63,12 @@ The following Jakarta EE 10 Liberty features do not expose the XML Binding 4.0 A
 
 For Open Liberty applications that require the XML Binding APIs, you must now add the feature:xmlBinding-4.0[display=Jakarta XML Binding 4.0] (`xmlBinding-4.0`) Liberty feature to your `server.xml` file.
 
+=== Admin REST Connector availability 
+
+The feature:jakartaee-9.1[display=Jakarta EE Platform 9.1] (`jakartaee-9.1`) Liberty convenience feature includes the feature:restConnector-2.0[display=Admin Rest Connector 2.0 ] (`restConnector-2.0`) Liberty feature. The feature:jakartaee-10.0[display=Jakarta EE Platform 10.0] (`jakartaee-10.0`) Liberty convenience feature does not include this Liberty feature.
+
+If you use the `jakartaee-10.0` convenience feature and your application requires the Admin Rest Connector function, you must add the `restConnector-2.0` feature to your `server.xml` file.
```

Comment:
@jhanders34 - I've edited your suggestion slightly to meet ID sentence length guidelines. Let me know if this works